### PR TITLE
Valgrind

### PR DESCRIPTION
**Pull request by wiredfool**
Run all of the tests in valgrind. 

* We're not checking for memory leaks.  (yet)
* This catches OOB Reads/Writes, and use of uninitialized memory. 
This takes ~1/2 hour on my system, so we should probably run this on .c/.h changes

### CODE REVIEW

File: ubuntu-20.04-focal-amd64-valgrind/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+RUN cd /depends \
+    && ./install_imagequant.sh \
+    && ./install_raqm.sh
+
+COPY python.supp /depends
+
+USER pillow
+CMD ["depends/test.sh"]
+
+#docker run -v $TRAVIS_BUILD_DIR:/Pillow pythonpillow/ubuntu-20.04-focal-amd64
```
Comment (radarhere) [inline ubuntu-20.04-focal-amd64-valgrind/Dockerfile:54]:
```suggestion
#docker run -v $TRAVIS_BUILD_DIR:/Pillow pythonpillow/ubuntu-20.04-focal-amd64-valgrind
```

Comment (wiredfool) [reply ubuntu-20.04-focal-amd64-valgrind/Dockerfile:54]: The little things that I do to keep some memory on how to run stuff manually, and the life that they lead later...

(and that was likely obsoleted by getting the makefile running, and furthermore: Travis? 

https://github.com/python-pillow/docker-images/search?q=travis

)

**Comment by wiredfool** [comment]
Can we push this with the known failure on the stable branch?  (Which, TBH is kind of the point of this)

**Comment by radarhere** [comment]
I don't think I mind failing stable - they're new jobs, so it's not exactly a step backwards.

File: ubuntu-20.04-focal-amd64-valgrind/Dockerfile (hunk 1 above)
Comment (radarhere) [reply ubuntu-20.04-focal-amd64-valgrind/Dockerfile:54]: #106 has updated the Travis reference.

**Comment by wiredfool** [comment]
Looks like Arch is failing with a semi known recent error: https://stackoverflow.com/questions/66154574/archlinux-docker-ci-failed-to-initialise-alpm-library-returned-a-non-zero-cod

**Pull request merged**